gauntlet/packages/gauntlet-solana/src/commands/internal/solana.ts

```diff
@@ -107,7 +143,41 @@ export default abstract class SolanaCommand extends WriteCommand<TransactionResp
     }
     const signedTx = await this.wallet.signTransaction(tx)
     logger.loading('Sending tx...')
-    return await sendAndConfirmRawTransaction(this.provider.connection, signedTx.serialize())
+    try {
+      return await sendAndConfirmRawTransaction(this.provider.connection, signedTx.serialize())
+    } catch (error) {
+      // Retry mechanism with greater priority fees
+      if (
+        error instanceof SendTransactionError &&
+        error.message.includes('congestion') &&
+        validateRetryPriorityInput(this.flags.priorityRetry)
```

Comment:
same thing here. move validation inside of the if statement so you can provide a helpful failure error response if the input is not valid